services/brig/test/integration/Util.hs

```diff
@@ -135,6 +136,38 @@ type Spar = Request -> Request
 
 data FedClient (comp :: Component) = FedClient HTTP.Manager Endpoint
 
+-- | Note: Apply this function last when composing (Request -> Request) functions
+apiVersion :: ByteString -> Request -> Request
```

Comment:
A nit-pick: why not make the first argument a specific version, e.g., `V1` instead of a `ByteString`? When there will be a need to change use sites of `apiVersion`, we can grep for `apiVersion` in the project, but I thought it'd be even nicer if we used versions instead of their string representations.